src/css/HeaderContainer.css

```
@@ -1,10 +1,14 @@
 .header-container {
-  position: fixed;
-  top: 0;
-  left: 50%;
-  transform: translateX(-50%);
-  max-width: 480px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

텍스트 위치가 정 중앙 정렬이 될 것 같은데 피그마 보면 조금 아래에 위치해 있거든요
```suggestion
  display: flex;
  justify-content: center;
  align-items: baseline;
  padding: 1rem; # 이건 실제로 보면서 조정 필요
```

이렇게 수정하는 게 더 나아 보여요 ~

---

다시 수정해주세요